src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
it's especially complicated because it handles the old RUC 1.0 use case, and with and without babel plugins in 2.0. It's handling a lot of use cases. it's very abstract. i have plans to simplify it though. 

I dont understand how u cant just manually do it. Maybe just semi-manually do it. export `CHUNK_NAMES` from the package via  PR. I'll accept it. And then just call `wrapped.chunkName` in the importMessages file, and push it on the array, and voila.